engine/api/application_import.go

```diff
@@ -68,7 +69,8 @@ func (api *API) postApplicationImportHandler() Handler {
 		if globalError != nil {
 			myError, ok := globalError.(sdk.Error)
 			if ok {
-				return WriteJSON(w, msgListString, myError.Status)
+				log.Warning("postApplicationImportHandler> Unable to import application %s : %s", eapp.Name, myError.String())
+				return WriteJSON(w, myError.String(), myError.Status)
```

Comment:
humm no
we have to send msgListString because there can be useful message for the user.
Why not add the the error at the end of the list (with the right translation) 

btw: return the error should be enough because does a WriteJSON(...) of the error